modules/xml/src/test/java/org/mule/util/XmlLibrariesVerificationTestCase.java

```diff
@@ -58,4 +59,11 @@ public void transformer()
         TransformerFactory factory = TransformerFactory.newInstance();
         assertThat(factory, instanceOf(org.apache.xalan.processor.TransformerFactoryImpl.class));
     }
+
+    @Test
+    public void schema()
```

Comment:
It is generated from the SchemaLoader instance, so there is a direct dependency.